gems/sorbet-runtime/lib/types/types/typed_array.rb

```diff
@@ -60,6 +60,11 @@ def valid?(obj)
         obj.is_a?(Array)
       end
 
+      def freeze
+        type # force lazy initialization before freezing the object
```

Comment:
Should this be calling `build_type` instead?

Reply:
Indeed, it would be cleaner even if equivalent 👍